src/site/content/en/blog/install-thumbor/index.md

```diff
@@ -0,0 +1,184 @@
+---
+title: How to install the Thumbor image CDN
+subhead: Learn how to install the Thumbor image CDN on your own server.
+authors:
+  - katiehempenius
+date: 2019-07-29
+hero: hero.jpg
+alt: A pile of photos.
+description: |
+  Instructions on how to install Thumbor is an open-source image CDN and can be used for free to resize, compress, and transform images.
```

Comment:
Looks like you started writing one sentence and then finished with another: 

> Instructions on how to install Thumbor is an open-source image CDN

You can fix by deleting `Instructions on how to install`